README.md

```diff
@@ -1 +1,162 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+Install Ruby, that depends about what OS you have:
+
+[https://www.ruby-lang.org/en/downloads/](https://www.ruby-lang.org/en/downloads/)
+
+Install Git:
+
+[https://git-scm.com/downloads](https://git-scm.com/downloads)
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/grauJavier/catalog-of-my-things-ruby
+  cd catalog-of-my-things-ruby
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  ruby main.rb 
+```
+And start following the indications on the screen
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Dani Morillo**
+
+- GitHub: [danifromecuador](https://github.com/danifromecuador)
+- LinkedIn: [danifromecuador](https://www.linkedin.com/in/danifromecuador)
+
+👤 **Javier Grau**
+- GitHub: [grauJavier](https://github.com/grauJavier)
+- LinkedIn: [@javiergrau](https://www.linkedin.com/in/javiergrau/)
+- Twitter: [@jgrauchile](https://twitter.com/jgrauchile)
+
+👤 **Ali Baba**
+- GitHub: [Alibaba2023](https://github.com/Alibaba2023)
+- LinkedIn: [@ali-baba-hussaini-630607267](https://www.linkedin.com/in/ali-baba-hussaini-630607267/)
+- Twitter: [@AliBabaHu2023](https://twitter.com/AliBabaHu2023)
+
+👤 **Lawrence Muema**
+- GitHub: [Kidd254](https://github.com/Kidd254)
+- LinkedIn: [@lawrence-muema-kioko](https://www.linkedin.com/in/lawrence-muema-kioko/)
+- Twitter: [@lawrenc98789206](https://twitter.com/lawrenc98789206)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- **The user should be able to remove a selected book**
+- **The user will add genre to selected book**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/grauJavier/catalog-of-my-things-ruby/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give me an star on [my GitHub repo](https://github.com/grauJavier/catalog-of-my-things-ruby)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank to [The Odin Project](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby) for teaching me about Ruby
+and to [Microverse Inc.](https://www.microverse.org/) for doing things happen
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [Optional] please, your MIT link is broken. Kindly fix this to have a professional README file.
![Screenshot_26](https://github.com/grauJavier/catalog-of-my-things-ruby/assets/65298048/fd18bff9-c55d-41b4-8823-92e5fb4ebff8)